subsys/net/lib/lwm2m/lwm2m_obj_firmware_pull.c

```diff
@@ -38,6 +38,7 @@ static struct coap_block_context firmware_block_ctx;
 #define COAP2HTTP_PROXY_URI_PATH	"coap2http"
 
 static char proxy_uri[URI_LEN];
+static char *cfg_proxy_uri;
```

Comment:
I think we can assign `static char *cfg_proxy_uri = CONFIG_LWM2M_FIRMWARE_UPDATE_PULL_COAP_PROXY_ADDR` here.